### PR TITLE
Fix etcd connection error once a etcd down is back up

### PR DESCRIPTION
**Pull request by anmolbabu**
tendrl-bug-id: Tendrl/node-monitoring#70
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

**Comment by anmolbabu** [comment]
Needs to be verified

**Comment by anmolbabu** [comment]
Verified

**Comment by anmolbabu** [comment]
@shtripat @nthomas-redhat @r0h4n Please review

File: tendrl/node_monitoring/plugins/tendrl_ceph_cluster_iops.py [hunk 1]
```diff
@@ -66,7 +67,15 @@ def send_metric(
 def read_callback(data=None):
     global CONFIG
     try:
-        iops_read, iops_write = fetch_cluster_iops(CONFIG['cluster_name'])
+        iops_read, iops_write, err = fetch_cluster_iops(CONFIG['cluster_name'])
+        if err:
+            collectd.info(
```
Comment (r0h4n) [inline tendrl/node_monitoring/plugins/tendrl_ceph_cluster_iops.py]: This should be "collectd.error"

File: tendrl/node_monitoring/utils/central_store_util.py [hunk 2]
```diff
@@ -1,8 +1,16 @@
 from tendrl.commons.utils.time_utils import now as tendrl_now
 
 
+def write_etcd(key, value):
```
Comment (r0h4n) [inline tendrl/node_monitoring/utils/central_store_util.py]: Use https://github.com/Tendrl/commons/blob/develop/tendrl/commons/utils/etcd_utils.py#L42

File: tendrl/node_monitoring/plugins/tendrl_ceph_cluster_iops.py (hunk 1 above)
Comment (anmolbabu) [inline tendrl/node_monitoring/plugins/tendrl_ceph_cluster_iops.py]: Done

File: tendrl/node_monitoring/utils/central_store_util.py (hunk 2 above)
Comment (anmolbabu) [inline tendrl/node_monitoring/utils/central_store_util.py]: Done

**Comment by anmolbabu** [comment]
@r0h4n @nthomas-redhat @shtripat Please review

**Pull request merged**